src/core/init/initial_seek_and_play.ts

```diff
@@ -138,10 +138,12 @@ export default function seekAndLoadOnMediaEvents(
 ) : { seek$ : Observable<unknown>; load$ : Observable<ILoadEvents> } {
   const seek$ = whenLoadedMetadata$(mediaElement).pipe(
     take(1),
-    tap(() => {
-      log.info("Init: Set initial time", startTime);
-      mediaElement.currentTime = typeof startTime === "function" ? startTime() :
-                                                                   startTime;
+    mergeMap(() => {
```

Comment:
I suggest to use the `withLatestFrom` operator from RxJS here. You can take the latest value of the clock tick, without having to re-piping the `clock` into the `mergeMap`.

Reply:
Yep. And you would have no need to set `take(1)` either.
And it would probably be more readable

Reply:
Done